evm/src/cross_table_lookup.rs

```diff
@@ -580,42 +547,25 @@ pub(crate) fn verify_cross_table_lookups<
 >(
     cross_table_lookups: &[CrossTableLookup<F>],
     ctl_zs_lasts: [Vec<F>; NUM_TABLES],
-    degrees_bits: [usize; NUM_TABLES],
-    challenges: GrandProductChallengeSet<F>,
     config: &StarkConfig,
 ) -> Result<()> {
     let mut ctl_zs_openings = ctl_zs_lasts.iter().map(|v| v.iter()).collect::<Vec<_>>();
-    for (
-        i,
-        CrossTableLookup {
-            looking_tables,
-            looked_table,
-            default,
-            ..
-        },
-    ) in cross_table_lookups.iter().enumerate()
+    for CrossTableLookup {
+        looking_tables,
+        looked_table,
+    } in cross_table_lookups.into_iter()
     {
         for _ in 0..config.num_challenges {
-            let looking_degrees_sum = looking_tables
-                .iter()
-                .map(|table| 1 << degrees_bits[table.table as usize])
-                .sum::<u64>();
-            let looked_degree = 1 << degrees_bits[looked_table.table as usize];
             let looking_zs_prod = looking_tables
                 .iter()
                 .map(|table| *ctl_zs_openings[table.table as usize].next().unwrap())
                 .product::<F>();
             let looked_z = *ctl_zs_openings[looked_table.table as usize].next().unwrap();
-            let challenge = challenges.challenges[i % config.num_challenges];
-
-            if let Some(default) = default.as_ref() {
-                let combined_default = challenge.combine(default.iter());
-                ensure!(
```

Comment:
(I think this was the CTL verifier issue, we were only doing the check if a default was present)